torchci/lib/bot/pytorchBot.ts

```diff
@@ -58,13 +61,40 @@ function pytorchBot(app: Probot): void {
     async function handleConfused() {
       await reactOnComment(ctx, "confused");
     }
+
+    function isValidForceMergeMessage(message: string): boolean {
+      // We can enforce  the merge message format here, for example, rejecting
+      // all messages not in the following format `[CATEGORY] description`.
+      //
+      // However, it seems too strict to enforce a fixed set of categories right
+      // away without conducting a user study for all common use cases of force
+      // merge first. So the message is just a free form text for now
+      if (message?.match(forceMergeMessagePat)) {
+        return true;
+      }
+
+      return false;
```

Comment:
```suggestion
      return message?.match(forceMergeMessagePat);
```

Reply:
Fixed. This needs a bit of a tweaking to address type mismatch warning. Basically, `match` returns `RegExpMatchArray | null` and this cannot be explicitly converted to boolean. But using it in an if statement is fine. I need to boost my typescript skills :)

So the final version is as follows:
```
const matches = message?.match(forceMergeMessagePat);
return matches != undefined && matches.length != 0;
```